torchxrayvision/models.py

```diff
@@ -141,7 +141,8 @@ def __init__(self, growth_rate=32, block_config=(6, 12, 24, 16), num_init_featur
         
         if self.weights != None:
             if not self.weights in model_urls.keys():
-                raise Exception("weights value must be in {}".format(list(model_urls.keys())))
+                possible_weights = [k for k in model_urls.keys() if k.startswith("densenet")]
+                raise Exception("Weights value must be in {}".format(possible_weights))
```

Comment:
what about resnet50-res512-all?  maybe the criterion should be if len(k.split('-'))==3.  I'm assuming the pattern is that all that are fully specified arch-resolution-dataset can be loaded.